src/views/clusteroverview/OverviewTab/metric-charts-card/components/MetricChart.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { abbreviateNumber } from 'js-abbreviation-number';
+
+import useResponsiveCharts from '@kubevirt-utils/components/Charts/hooks/useResponsiveCharts';
+import {
+  dateFormatterNoYear,
+  timestampFor,
+} from '@kubevirt-utils/components/Timestamp/utils/datetime';
+import {
+  Chart,
+  ChartArea,
+  ChartAxis,
+  ChartGroup,
+  ChartVoronoiContainer,
+} from '@patternfly/react-charts';
+import chart_color_blue_100 from '@patternfly/react-tokens/dist/esm/chart_color_blue_100';
+
+import useDateTicks from '../utils/hooks/useDateTicks';
+import { MetricChartData } from '../utils/hooks/useMetricChartData';
+import { getLabeledTickIndexes, getLabelUnit } from '../utils/utils';
+
+import './MetricChart.scss';
+
+type MetricChartProps = { metricChartData: MetricChartData; metric: string };
+
+const MetricChart: React.FC<MetricChartProps> = ({ metricChartData, metric }) => {
+  const { chartData, domain, unit } = metricChartData;
+  const { ref, width, height } = useResponsiveCharts();
+  const displayUnit = getLabelUnit(metric, unit);
+  const ticks = useDateTicks(chartData);
+
+  return (
+    <div className="overview-metric-chart" ref={ref}>
+      <Chart
+        height={height}
+        width={width}
+        padding={{ top: 35, bottom: 35, left: 100, right: 10 }}
+        scale={{ x: 'time', y: 'linear' }}
+        domain={domain}
+        containerComponent={
+          <ChartVoronoiContainer
+            labels={({ datum }) => {
+              return `${timestampFor(
+                datum?.x as Date,
+                new Date(),
+                false,
+              )}\n ${datum?.y?.toLocaleString()} ${displayUnit}`;
+            }}
+            constrainToVisibleArea
+          />
+        }
+      >
+        <ChartAxis
+          dependentAxis
+          tickCount={2}
+          tickValues={[0, domain?.y?.[1]]}
+          tickFormat={(tick) => {
+            return `${abbreviateNumber(tick, 1)} ${getLabelUnit(metric, unit)}`;
+          }}
+          style={{
+            ticks: {
+              stroke: 'transparent',
+            },
+            grid: {
+              stroke: chart_color_blue_100.value,
+            },
+          }}
+          axisComponent={<></>}
+        />
+        <ChartAxis
+          tickCount={ticks.length}
+          tickValues={ticks}
```

Comment:
the ticks logic in the hooks seems like an overhead, I think it would be nice to have like in the y-axis just the two boudries of the x-axis, wdyt?

Reply:
The format of the x-axis tick labels was also discussed at length and the decision was made to label the first, middle, and final ticks with the last tick being labeled 'Today'.